src/main/kotlin/org/rust/ide/inspections/RsLint.kt

```diff
@@ -30,7 +31,8 @@ enum class RsLint(
     private fun explicitLevel(el: PsiElement): RsLintLevel? = el.ancestors
         .filterIsInstance<RsDocAndAttributeOwner>()
         .flatMap { it.queryAttributes.metaItems }
-        .filter { it.metaItemArgs?.metaItemList.orEmpty().any { it.name == id || it.name == groupId } }
+        .filter { it.metaItemArgs?.metaItemList.orEmpty()
+            .any { it.text == id || it.text in groupIds } }
```

Comment:
Maybe it's better to use `path?.text` instead of whole text here?
And maybe it's worth to extract `metaItem.path?.text` into some extension property